_posts/2021-03-09-gsoc2021.md

```diff
@@ -0,0 +1,118 @@
+page — we might add more ideas after the posting date of this post.
+
+You don't need to have all the skills that we are listing, although
+that helps, of course. But you need to demonstrate to us that you're
+able and keen to learn anything that you don't know yet, and we will
+be happy to help you learn during your project with us.
+
+| project | name                                                                                                                                                      | difficulty | description                                                                                    | skills                      | mentors                                |
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------------------------------------------------------------------------------------------|-----------------------------|----------------------------------------|
+| 1       | [Molecular volume and surface analysis]({{ site.github.wiki }}/Project-Ideas-2021#project-1-molecular-volume-and-surface-analysis) | easy       | use an existing package for molecular surface area calculations to build a new analysis module | Python, MDAnalysis.analysis | @orbeckst, @IAlibay  |
+| 2       | [AtomGroupGroup and Rings]({{ site.github.wiki }}/Project-Ideas-2021#project-2-atomgroupgroup-and-rings)                           | medium     | Write analysis module for pi stacking analysis                                                 | Python                      | @richardjgowers, @micaela-matta, @IAlibay        |
+| 3       | [Generalise Groups]({{ site.github.wiki }}/Project-Ideas-2021#project-3-generalise-groups)                                         | medium     | Generalise concept of groups                                                                   | Python                      | @lilyminium, @fiona-naughton (?) |
+| 4       | [Cythonisation of AtomGroup]({{ site.github.wiki }}/Project-Ideas-2021#project-4-cythonise-atomgroup)                              | easy      | Cythonise AtomGroup for use with C/++                                                          | Python, Cython, C/++        | @richardjgowers, @hmacdope, @lilyminium (maybe)|
+| 5       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/Project-Ideas-2021#project-5-extend-interoperability)                  | medium     | Extend converters module to other relevant packages                                            | Python                      | @lilyminium, @IAlibay  |
+| 6       | [General unit cell representation]({{ site.github.wiki }}/Project-Ideas-2021#project-6-general-unit-cell-representation)                  | easy     |  change unit cell representation to keep track of box rotations and so improve analysis of simulations under periodic boundary conditions | Python                      | @orbeckst  |
```

Comment:
We probably need an extra mentor officially down here.

Reply:
Yes, if no one wants to mentor then we better strike the project.

Reply:
But note that for student-initiated projects there's also no guarantee that two mentors are enthusiastic. Although I agree that for anything _we_ offer we need to be clear that we would be mentoring.

Reply:
> Yes, if no one (EDIT) _else_ wants to mentor then we better strike the project.

@fiona-naughton would you be interested? (going off your discord comment)

Reply:
In the normal GSOC application process where they introduce themselves on the dev-list, I think there's room to make clear how much interest we have in their suggested project. The current text only says that we will discuss student-initiated projects, not that we will mentor them.

Reply:
Yeah, I'm happy to mentor here (I did add myself on the wiki but only very recently).